cmd/otelcorecol/go.mod

```
@@ -4,7 +4,7 @@ module go.opentelemetry.io/collector/cmd/otelcorecol
 
 go 1.21.0
 
-toolchain go1.21.11
+toolchain go1.22.4
```

The change here should be reverted

```suggestion
toolchain go1.21.11
```